lib/methwhy.g

```diff
@@ -305,17 +305,19 @@ end);
 BIND_GLOBAL("ShowImpliedFilters",function(fil)
 local flags,f,i,j,l,m,n;
   flags:=FLAGS_FILTER(fil);
-  f:=Filtered(IMPLICATIONS,x->IS_SUBSET_FLAGS(x[2],flags));
-  l:=[];
-  m:=[];
-  for i in f do
-    n:=SUB_FLAGS(i[2],flags); # the additional requirements
-    if SIZE_FLAGS(n)=0 then
-      Add(l,i[1]);
-    else
-      Add(m,[n,i[1]]);
-    fi;
-  od;
+  atomic readonly IMPLICATIONS do
```

Comment:
I'd like to explicitly point out that this adds another `atomic` statement to `lib`, which may be controversial.